src/types/protocols-types.ts

```diff
@@ -119,6 +119,7 @@ export type ProtocolRuleSet = {
 };
 
 export type ProtocolsConfigureMessage = GenericMessage & {
+  authorization: AuthorizationModel; // overriding `GenericMessage` with `authorization` being required
```

Comment:
is this an opportunistic change or is it necessary for the PR somehow? no pushback, just asking for my own understanding.

Reply:
Necessary, I realized that this condition is never hit in real code path (except by a unit test, which I also removed):

https://github.com/TBD54566975/dwn-sdk-js/pull/531/files/2c79913beec68c5a7f846cd749de0061182c7357#diff-ebe8b6ad983e00a9df763de521756d085f755da1cf9819d13766a1c84d485d22L36

The reason we had it was purely because of ambiguous typing, thus the "fix".